tests/tests/fv3_ccpp_gfs_v15p2_coldstart

```diff
@@ -0,0 +1,48 @@
+###############################################################################
+#
+#  FV3 CCPP GFS v15.2 compiled with 32-bit dynamics coldstart (0-24h) test
+#
+###############################################################################
+
+export TEST_DESCR="Compare FV3 32bit CCPP GFS v15.2 results with previous trunk version"
+
+export CNTL_DIR=fv3_gfs_v15p2
```

Comment:
Is this correct CNTL_DIR? Shouldn't it be fv3_gfs_v15p2_ccpp?

Reply:
The CNTL_DIR is correct, the run script appends the RT_SUFFIX environment variable so that this test works for IPD PROD, CCPP PROD, IPD/CCPP REPRO.

Reply:
I see. Thanks.